backend/src/main/java/com/cgi/retrospecto/backend/poker/service/impl/RoomServiceImpl.java

```diff
@@ -0,0 +1,85 @@
+package com.cgi.retrospecto.backend.poker.service.impl;
+
+import com.cgi.retrospecto.backend.poker.domain.Room;
+import com.cgi.retrospecto.backend.poker.domain.Story;
+import com.cgi.retrospecto.backend.poker.domain.Vote;
+import com.cgi.retrospecto.backend.poker.domain.VoteResult;
+import com.cgi.retrospecto.backend.poker.exception.RoomNotFoundException;
+import com.cgi.retrospecto.backend.poker.exception.StoryNotFoundException;
+import com.cgi.retrospecto.backend.poker.repository.RoomRepository;
+import com.cgi.retrospecto.backend.poker.service.RoomService;
+import com.cgi.retrospecto.backend.util.Helper;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import java.time.LocalDateTime;
+
+@Service
+public class RoomServiceImpl implements RoomService {
+
+    @Autowired
+    private RoomRepository repo;
+
+    @Override
+    public Room createRoom(Room room) {
+        room.setId(Helper.generateId(repo.getPokerRoomIds()));
+        room.setLastActionSubmittedTime(LocalDateTime.now());
+        repo.addRoom(room);
+        return room;
+    }
+
+    @Override
+    public Story createStory(int roomId, Story story) throws RoomNotFoundException {
+        final Room room = repo.getPokerRoom(roomId);
+        if (room == null) {
```

Comment:
There is already an exception thrown in the repo with getPokerRoom() so this null check is redundant.

Reply:
Removed redundant part.